docker-compose.yml

```diff
@@ -5,5 +5,7 @@ services:
     build:
       context: .
       dockerfile: Dockerfiles/Dockerfile.static_site
+    environment:
+      - JEKYLL_ENV=dev
```

Comment:
Do we need to update our deployment script to set `JEKYLL_ENV` appropriately?  For example: https://github.com/CMSgov/bcda-ops/blob/master/jenkins_files/Jenkinsfile.deploy_static_site#L71-L72

Reply:
Nevermind!  I just saw this PR with the appropriate change: https://github.com/CMSgov/bcda-ops/pull/600